q2_types/feature_table/_transformer.py

```diff
@@ -12,26 +12,36 @@
 import pandas as pd
 import qiime
 
-from . import BIOMV1Format
+from . import BIOMV1Format, BIOMV210Format
 from ..plugin_setup import plugin
 
 
+def _get_generated_by():
+    return 'qiime %s' % qiime.__version__
+
+
 @plugin.register_transformer
 def _1(data: biom.Table) -> BIOMV1Format:
+    # TODO: issue a warning about limited support for this format
```

Comment:
I think this TODO should be removed. It isn't the plugin's responsibility to warn users about older file formats, and users shouldn't be aware (or care) about what file format is used under the hood for an artifact. We will continue to support BIOM v1 for backward-compatibility with existing data, and users shouldn't be warned if their data is in that format as that will cause unnecessary confusion.

If we were to deprecate/warn about older file formats, it would be the framework's responsibility to handle this in a generic way. We don't currently have plans to do this, though, since there will be many older formats (not just related to BIOM) that we'll continue to support.